checkov/main.py

```diff
@@ -40,9 +40,9 @@ def run():
     if args.list:
         print_checks()
         return
-    else:
```

Comment:
why remove the else?

Reply:
@schosterbarak Because in the case of the `if`, there is a `return`. So the else does nothing

Reply:
missed that 👍